homeassistant/helpers/entity.py

```diff
@@ -183,8 +183,8 @@ def device_info(self) -> Optional[Dict[str, Any]]:
         return None
 
     @property
-    def device_class(self) -> Optional[str]:
-        """Return the class of this device, from component DEVICE_CLASSES."""
+    def entity_class(self) -> Optional[str]:
+        """Return the class of this entity, from component DEVICE_CLASSES."""
```

Comment:
```suggestion
        """Return the class of this entity, from component ENTITY_CLASSES."""
```

Reply:
Yes, but `ENTITY_CLASSES` do not yet exist, maybe hold this until the base components are updated?